api/v1/middleware/auth.js

```diff
@@ -17,12 +20,27 @@ module.exports = (req, res, next) => {
       // specifies that request supplied a valid auth token
       // (but not necessarily that the associated user data has been retrieved)
       req.auth = true;
-
       return User.findById(decoded.sub);
     })
     .then((user) => {
-      req.user = user;
+      const adminUserOverride = req.get('Admin-User-Override');
+
+      if (!_.isUndefined(adminUserOverride) && PermissionService.isOrganizer(user)) {
```

Comment:
This should probably be just admins (not organizers). Otherwise there's really little distinction in what's going on (most members of our org are admins or staff).